x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality_panel/data_quality_details/indices_details/pattern/hooks/use_ilm_explain/index.test.tsx

```diff
@@ -89,10 +90,10 @@ describe('useIlmExplain', () => {
     beforeEach(async () => {
       mockHttpFetch.mockResolvedValue(mockIlmExplain);
 
-      const { result, waitForNextUpdate } = renderHook(() => useIlmExplain(pattern), {
+      const { result } = renderHook(() => useIlmExplain(pattern), {
```

Comment:
Please use `waitFor` from renderHook, it's different from `@testing-library/react` implementation-wise, I assume that's why they put a separate version of it in the exports from renderHook

```suggestion
      const { result, waitFor } = renderHook(() => useIlmExplain(pattern), {
```

Reply:
I was wrong the `renderHook` from rtl for v18 removes `waitFor` from return https://github.com/testing-library/react-testing-library/blob/3dcd8a9649e25054c0e650d95fca2317b7008576/src/pure.js#L310